datatypes/papas/Cluster.h

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
Should be in papas/datatypes/Cluster.h, not datatypes/papas/Cluster.h I think. 
It seems that's part of the todo list, so I won't comment for the other files